pkg/apis/autoscaling/v1/verticalpodautoscalercontroller_types.go

```diff
@@ -17,7 +18,38 @@ type VerticalPodAutoscalerSpec struct {
 	// +kubebuilder:validation:Minimum=0
 	PodMinMemoryMb     *float64 `json:"podMinMemoryMb,omitempty"`
 	RecommendationOnly *bool    `json:"recommendationOnly,omitempty"`
-	MinReplicas        *int64   `json:"minReplicas,omitempty"`
+	// +kubebuilder:validation:Minimum=1
```

Comment:
Consider mentioning in the commit message why this validation is needed.

Reply:
I'm not actually changing it, the old CRD had it set, but it was decoupled from the types and I wanted it retained in the CRD generation for compatibility, I'm just truing up the generation with the CRD customers have been using.